internal/app/app.go

```diff
@@ -205,6 +205,9 @@ func (app *App) doStuff(files []string, variables map[string]interface{}) error
 				return err
 			}
 			configType := app.Env.GetConfigType(files[i])
+			if configType == "unknown" {
+				continue
```

Comment:
should it print a warning here with the filename if the config is unknown?

Reply:
Yeah we should. I added that check before making sure it only looked in the right directories. I was afraid of too much logs for files we don't care about, but I guess that won't be an issue now. I'll add it